src/client/components/AddNewRoom/AddNewRoom.stories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+import RoomHandler from './RoomHandler';
```

Comment:
Is this a story for `AddNewRoom` as it says in the title or is it for the `RoomHandler`?
The convention is to ensure file names are in sync.